spark/src/main/scala/org/apache/mahout/sparkbindings/blas/ABt.scala

```diff
@@ -116,7 +118,12 @@ object ABt {
           // Empty combiner += value
           createCombiner = (t: (Array[K], Array[Int], Matrix)) =>  {
             val (rowKeys, colKeys, block) = t
-            val comb = new SparseMatrix(prodNCol, block.nrow).t
```

Comment:
thanks.  Well there is really no rush to do this half way, so I'll just create a JIRA for sparse/dense in memory analysis, and get it done completely.